standard_labels.mss

```diff
@@ -40,10 +40,10 @@
 
 #railway_text_stations_med[zoom>=6][zoom<=10],
 #railway_text_stations_high[zoom>=11] {
-  [railway = 'station'][station != 'light_rail'][station != 'subway'][zoom <= 9],
+  [railway = 'station'][station != 'light_rail'][station != 'subway'][station != 'funicular'][zoom <= 9],
   [railway = 'tram_stop'][zoom >= 14],
-  [railway = 'station'][zoom >= 10],
-  [railway = 'halt'][zoom >= 10],
+  [railway = 'station'][station != 'funicular'][zoom >= 10],
+  [railway = 'halt'][station != 'funicular'][zoom >= 10],
   [railway != 'tram_stop'][zoom >= 11] {
```

Comment:
Your pull request does still render the stations. Please add `[station != 'funicular']` in this line as well. Otherwise the layer `railway_text_station_high` will look this way in the resulting XML file using the Carto transpiler:

```xml
  <Style filter-mode="first" name="railway_text_stations_high">
    <Rule>
      <MaxScaleDenominator>400000</MaxScaleDenominator>
      <Filter><![CDATA[([railway] = 'halt') and ([station] != 'funicular')]]></Filter>
      <TextSymbolizer fill="#0000ff" fontset-name="fontset-0" halo-fill="#ffffff" halo-radius="1.5" minimum-distance="30" minimum-padding="10" size="12" wrap-width="50"><![CDATA[[label]]]></TextSymbolizer>
    </Rule>
    <Rule>
      <MaxScaleDenominator>400000</MaxScaleDenominator>
      <Filter><![CDATA[([railway] = 'station') and ([station] != 'funicular')]]></Filter>
      <TextSymbolizer fill="#0000ff" fontset-name="fontset-0" halo-fill="#ffffff" halo-radius="1.5" minimum-distance="30" minimum-padding="10" size="12" wrap-width="50"><![CDATA[[label]]]></TextSymbolizer>
    </Rule>
    <Rule>
      <MaxScaleDenominator>400000</MaxScaleDenominator>
      <Filter><![CDATA[([railway] = 'yard')]]></Filter>
      <TextSymbolizer fill="#87491d" fontset-name="fontset-0" halo-fill="#f1f1f1" halo-radius="1.5" minimum-distance="30" minimum-padding="10" size="12" wrap-width="50"><![CDATA[[label]]]></TextSymbolizer>
    </Rule>
    <Rule>
      <MaxScaleDenominator>50000</MaxScaleDenominator>
      <Filter><![CDATA[([railway] = 'tram_stop')]]></Filter>
      <TextSymbolizer dy="9" fill="#d877b8" fontset-name="fontset-0" halo-fill="#ffffff" halo-radius="1.5" minimum-distance="30" minimum-padding="10" size="12" wrap-width="50"><![CDATA[[label]]]></TextSymbolizer>
    </Rule>
    <Rule>
      <MaxScaleDenominator>400000</MaxScaleDenominator>
      <Filter><![CDATA[([railway] = 'halt')]]></Filter>
      <TextSymbolizer fill="#0000ff" fontset-name="fontset-0" halo-fill="#ffffff" halo-radius="1.5" minimum-distance="30" minimum-padding="10" size="12" wrap-width="50"><![CDATA[[label]]]></TextSymbolizer>
    </Rule>
    <Rule>
      <MaxScaleDenominator>400000</MaxScaleDenominator>
      <Filter><![CDATA[([railway] = 'station')]]></Filter>
      <TextSymbolizer fill="#0000ff" fontset-name="fontset-0" halo-fill="#ffffff" halo-radius="1.5" minimum-distance="30" minimum-padding="10" size="12" wrap-width="50"><![CDATA[[label]]]></TextSymbolizer>
    </Rule>
    <Rule>
      <MaxScaleDenominator>400000</MaxScaleDenominator>
      <Filter><![CDATA[([railway] != 'tram_stop')]]></Filter>
      <TextSymbolizer fill="#616161" fontset-name="fontset-0" halo-fill="#f1f1f1" halo-radius="1.5" minimum-distance="30" minimum-padding="10" size="12" wrap-width="50"><![CDATA[[label]]]></TextSymbolizer>
    </Rule>
  </Style>
```

Reply:
Yes, I misread the last selection (!= vs. =).